content/_index.md

```diff
@@ -109,4 +109,6 @@ _This research/project is supported by the Australian Research Data Commons
 
 </div>
 
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7743217.svg)](https://doi.org/10.5281/zenodo.7743217)
```

Comment:
ok, for some reason it looks like the image badge is being stripped:

![image](https://user-images.githubusercontent.com/1452968/226216403-cc1da81d-1550-48bc-93ff-4dcf98a45c0a.png)

it would probably be better to put this info in the footer so it is shown on every page

Reply:
I removed the bagde from the front page and moved it to the footer, 

![image](https://user-images.githubusercontent.com/46727219/227446350-7824987d-bdd0-4cba-886b-5869e869b34c.png)